core/src/main/java/com/datastrato/gravitino/Configs.java

```diff
@@ -157,6 +160,19 @@ public interface Configs {
           .longConf()
           .createWithDefault(2000L);
 
+  @Deprecated
```

Comment:
The configuration option has a method `deprecated` instead of annonatioin.

Reply:
> The configuration option has a method `deprecated` instead of annonatioin.

Thx.